src/pages/developing/svelte-tutorial/step-2.mdx

```diff
@@ -0,0 +1,719 @@
+    </Column>
+  </Row>
+</TabContent>
+```
+<!-- prettier-ignore-end -->
+
+Now let's set the image size in the `style` block of the file:
+
+```html path=src/routes/index.svelte
+<style>
+  img {
+    max-width: 100%;
+  }
+</style>
+```
+
+Assuming that the second and third tab would have a similar design, we would set
+them up in the same way. However, since our design specs don't show those tabs,
+we'll leave the code as is.
+
+### Third row
+
+The third row will be created in a later tutorial, so we'll just add the headers
+for now.
+
+```html path=src/routes/index.svelte
+<Row class="landing-page-background landing-page__r3">
+  <Column md="{4}" lg="{4}">
+    <h3>The Principles</h3>
+  </Column>
+  <Column md="{4}" lg="{4}">Carbon is Open</Column>
+  <Column md="{4}" lg="{4}">Carbon is Modular</Column>
+  <Column md="{4}" lg="{4}">Carbon is Consistent</Column>
+</Row>
+```
+
+### Banner
+
+<Row>
+<Column colLg={8}>
+
+![Banner vertical spacing](../shared/step-2/images/landing-r1-spacing.png)
+
+<Caption>Banner vertical spacing</Caption>
+
+</Column>
+</Row>
+
+<InlineNotification>
+
+**Pro tip:** `CTRL-G` toggles the grid in Sketch.
+
+</InlineNotification>
+
+Back to the `style` block, we need to add space above the breadcrumb and below
+the heading. For that, add:
+
+```css path=src/routes/index.svelte
+:global(.landing-page__banner) {
+  padding-top: 'spacing-05';
+  padding-bottom: 'spacing-07' * 4;
+}
+```
+
+We need to use the `:global(...)` modifier because the `landing-page__banner`
+class passed to the `Row` component is out of scope from the current file. Thus,
+we must target it globally.
+
+Referencing the
+[spacing token table](https://www.carbondesignsystem.com/guidelines/spacing/overview#spacing-scale),
+`16px` can be set with the `$spacing-05` token. The design calls for `128px` of
+space below the heading and that's not in the spacing scale. We can achieve this
+in Sass by multiplying 32px (`$spacing-07`) by 4. We could use `128px` or `8rem`
+directly in our styling, but using our tokens preserves consistency should the
+token values get updated in the future.
+
+Looking at the design, we need a wall-to-wall light gray background behind the
+banner and also behind the third row.
+
+Per the design we need to use Gray 10 for our banner background color, which can
+be set with the `$ui-01`
+[color token](https://www.carbondesignsystem.com/guidelines/color/usage). Also,
+we want the background to extend into the grid's outermost gutters to go the
+full width of the viewport, so given the DOM structure, we can achieve that by
+setting the background in an absolutely positioned pseudo element.
+
+```css path=src/routes/index.svelte
+:global(.landing-page-background) {
+  position: relative;
+  background-color: 'ui-01';
+}
+
+:global(.landing-page-background::before) {
+  content: '';
+  position: absolute;
+  z-index: -1;
+  top: 0;
+  bottom: 0;
+  right: -'spacing-05';
+  left: -'spacing-05';
+  background: 'ui-01';
+}
+```
```

Comment:
There is a visual difference in the grid background span, seems to be these styles which are included in https://svelte-step-3--carbon-tutorial-svelte.netlify.app/ but not in the tutorial:
<img width="247" alt="image" src="https://user-images.githubusercontent.com/40550942/201173951-429b3da3-e4b2-4014-aac2-fda1301b6965.png">

Deployed Step 2:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/201174024-0b9755d1-6de8-411f-afca-7db7cea1a4af.png">

Result from following tutorial:
<img width="1792" alt="image" src="https://user-images.githubusercontent.com/40550942/201174174-aa21d98f-0012-4824-a228-43bfb6708f4b.png">